Integrated in latest BR.

Signed-off-by: Jeremy Kerr
Signed-off-by: Thomas Petazzoni

Changeset:
Modified: buildroot/target/makedevs/makedevs.c
--- buildroot/target/makedevs/makedevs.c	2008-11-03 09:15:04.000000000 +0100
+++ buildroot/target/makedevs/makedevs.c	2008-12-03 11:23:20.000000000 +0100
@@ -528,7 +528,8 @@ loop:
 	}
 	fclose(table);
 
-	system("/bin/sync");
+	if (system("/bin/sync"))
+		bb_error_msg("sync failed, continuing anyway");
 
 	return 0;
 }
